pkg/util/pretty/util.go

```diff
@@ -133,6 +133,14 @@ func FoldMap(f func(a, b Doc) Doc, g func(Doc) Doc, d ...Doc) Doc {
 
 // Bracket brackets x with l and r and given Nest arguments.
 func Bracket(n int, l string, x Doc, r string) Doc {
+	// The "straightforward" implementation of Bracket should really be:
```

Comment:
This comment isn't accurate. The reason we can't use Group is because we need `Line` to be the empty string if it is flattened, instead of the normal `" "`.

Reply:
Hugh, this was totally invisible to me. It would have helped if that was documented :)

I'll add the correct comment.

Reply:
Done:  #27263